android/src/main/java/org/reactnative/camera/CameraModule.java

```diff
@@ -369,4 +373,22 @@ public void execute(NativeViewHierarchyManager nativeViewHierarchyManager) {
             }
         });
     }
+
+    @ReactMethod
+    public void checkIfRecordAudioPermissionsAreDefined(final Promise promise) {
```

Comment:
I added a new method similar to the one on ios to check if the key is defined in the `AndroidManifest.xml` before asking for the permissions.